src/themes/bankrate/theme.json

```diff
@@ -764,6 +764,26 @@
           ":active": {
             "backgroundColor": "button-secondary--hover"
           }
+        },
+        "landing-page": {
```

Comment:
I think we should call this a name that represents what it is, not where it's used. So, `icon` or something